### DISCUSSION TITLE
Links to non-markdown files in vault

### DISCUSSION THREAD

**Discussion by wtaverni** [oleeskild/obsidian-digital-garden#117] [Q&A]
How can I link to another file in my vault (e.g. a PDF file)? If I use the format `[link](file.pdf)`, the file is not found on the online digital garden. This makes sense, as there is no way to have a yaml frontmatter for a PDF file including dg-publish: true.

Is there an easy way to publish other files which are not .md files?

**Comment by oleeskild** [comment]
This is not supported yet. It would be nice to have an option to publish arbitrary files via a right-click menu or something. You could make a Feature request under [Issues](https://github.com/oleeskild/obsidian-digital-garden/issues). 

For now, what you can do, is upload it to some cloud storage like Dropbox, Google Drive or iCloud, make it public, and then use a normal markdown link to that url. Something like` [My Pdf](https://drive.google.com/......)`.

**Reply by wtaverni** [reply]
OK, thanks for your response! Indeed, linking to Dropbox is what I did!

**Comment by dmje** [comment]
I'd love for this to be a thing. For me it's embedded mp3 files.

**Comment by benf2004** [comment]
I would also love if this could be a thing
